### DISCUSSION TITLE
Emby LXC is defaulted to privileged - is it OK to run as unprivileged?

### DISCUSSION THREAD

**Discussion by dmshimself** [tteck/Proxmox#738] [General]
I've been using the EMBY container very well for a while and thanks very much to tteck for providing it.  While reviewing all my containers I noticed that this one is privileged, so I wondered if anyone has comments on running it as unprivileged?

(I appreciate I could backup and restore and flick it over, so really I'm seeking options rather than just giving that a go!)

**Comment by tteck** [comment]
Running unprivileged is ok if you don't use any hardware acceleration.

**Reply by dmshimself** [reply]
Many thanks :-)

**Reply by CvH** [reply]
I am pretty sure that hw acceleration is possible with a unprivileged container, it works for other hw pretty well so there should be no reason that it won't work. Google results sounds promising too.
